motion/los_guidance/scripts/los_guidance_euler.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 # Written by Kristoffer Rakstad Solberg, Student
+# Documented by Christopher Strom and Jae Hyeong Hwang
```

Comment:
Changed name, Christopher?

Reply:
Saw from the latest commits that this was to prevent errors with ASCII, but shouldn't that be changed in every file then?

Reply:
This is only a problem with the .py sources, since python is interpreted. The problem can be solved by adding # -*- coding: utf-8 -*- to line 2, but changing ø -> o is less work, and I'm lazy :wink: